app/controllers/application_controller.rb

```diff
@@ -73,7 +73,7 @@ def current_user
 
   def current_admin_session
     return @current_admin_session if defined?(@current_admin_session)
-    @current_admin_session = AdminSession.find
+    @current_admin_session = Rails.cache.fetch("admin_settings"){AdminSetting.first}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.